magento/magento2
# Backend: User Role Checkbox alignement.

### Description (*)

Backend: User Role Checkbox alignement.
### Fixed Issues (if relevant)

This is direct pull request as per Magento reviewer suggested,

### Manual testing scenarios (*)


1.  Login into admin panel > System > User Roles > Click on Adminstrator User > Role User > Look checkbox alignment.

2. Actual Result
![actual-result](https://user-images.githubusercontent.com/43638601/51070192-5aa75580-1663-11e9-8044-bc7248825fc5.png)

3. Expected Result
![expected-result](https://user-images.githubusercontent.com/43638601/51070195-698e0800-1663-11e9-9211-8dfe9263ab6c.png)


### Contribution checklist (*)
